### PR TITLE
docs: add dyte docs to showcase

### PR DESCRIPTION
**Pull request by vaibhavshn**
## Motivation

Adding our docs to your showcase!

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

YES

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

NONE

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

NONE

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -460,6 +460,14 @@ const Users: User[] = [
     source: 'https://github.com/facebook/draft-js',
     tags: ['opensource', 'meta'],
   },
+  {
+    title: 'Dyte',
+    description: 'The most developer friendly live video SDK',
+    preview: require('./showcase/dyte.png'),
+    source: null,
+    website: 'https://docs.dyte.io',
+    tags: ['product', 'design', 'versioning', 'large']
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:469]:
```suggestion
    tags: ['favorite', 'product', 'design', 'versioning', 'large']
```

**Comment by Josh-Cena** [comment]
Thanks! Very nice site. Just waiting for the CLA check

**Comment by vaibhavshn** [comment]
> Thanks! Very nice site. Just waiting for the CLA check... It's a shame that we can't see the source though

Thanks! Signing the Employer CLA in a while, might take some time. Will update as soon as I do it.

We are planning on open sourcing it in the future though.

**Comment by Josh-Cena** [comment]
That's great to hear! Please remember to update the showcase data once you open-sourced your site. It would be an inspiration for other users 😉

**Comment by vaibhavshn** [comment]
> That's great to hear! Please remember to update the showcase data once you open-sourced your site. It would be an inspiration for other users 😉

Definitely!

**Comment by slorber** [comment]
❤️ great site

Please report customization use-cases here so that we can allow you to make those customizations in a more robust way:
https://github.com/facebook/docusaurus/discussions/5468

**Comment by vaibhavshn** [comment]
The CLA has been signed.

**Comment by vaibhavshn** [comment]
> ❤️ great site
> 
> Please report customization use-cases here so that we can allow you to make those customizations in a more robust way: #5468

Thanks!
Will collect points and post it in the discussion.

**Comment by slorber** [comment]
Thanks ;)

**Pull request merged**